src/analysis/special_functions/trigonometric/chebyshev.lean

```diff
@@ -4,24 +4,52 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Johan Commelin
 -/
 import analysis.complex.basic
-import ring_theory.polynomial.chebyshev
 import data.complex.exponential
+import data.polynomial.algebra_map
+import ring_theory.polynomial.chebyshev
 
 /-!
 # Multiple angle formulas in terms of Chebyshev polynomials
 
-* `polynomial.chebyshev.T_complex_cos`: the `n`-th Chebyshev polynomial evaluates on `complex.cos θ`
-  to the value `complex.cos (n * θ)`.
+This file gives the trigonometric characterizations of Chebyshev polynomials, for both the real and
+complex cosine.
 -/
 
 namespace polynomial.chebyshev
+open polynomial
+
+variables {R A : Type*} [comm_ring R] [comm_ring A] [algebra R A]
+
+@[simp] lemma T_aeval (x : A) (n : ℕ) : aeval x (T R n) = (T A n).eval x :=
+by rw [aeval_def, eval₂_eq_eval_map, map_T]
+
+@[simp] lemma U_aeval (x : A) (n : ℕ) : aeval x (U R n) = (U A n).eval x :=
+by rw [aeval_def, eval₂_eq_eval_map, map_U]
```

Comment:
These look like they should be called `aeval_T` and `aeval_U` to me, to match `map_T` and `map_U`.

Reply:
Same for all the new lemmas.